src/Gui/DlgPreferencesImp.cpp

```diff
@@ -471,6 +477,34 @@ void DlgPreferencesImp::applyChanges()
     }
 }
 
+void DlgPreferencesImp::isRebootRequired()

```

Comment:
The function name is confusing because one would assume it's a getter function but it does something and doesn't return a value. Also it doesn't _reboot_ but restarts the application.